### DISCUSSION TITLE
No models are loaded

### DISCUSSION THREAD

**Discussion by gerroon** [longy2k/obsidian-bmo-chatbot#79] [Q&A]
I used BMO for a while with my local Ollama. It used to work great, but I didn't need it for a couple of months. Now I have a need for it, but it lists no models. Not sure if an update broke it or not.

I use the exact same Ollama URL for all my local LLM apps; they all work at the moment.

"http://192.168.199.1:11434"

I am on Windows 11 Obsidian 1.5.12

**Comment by twalderman** [comment]
add /v1 to your path

**Reply by gerroon** [reply]
Thanks but it still does not work :(

**Comment by twalderman** [comment]
Im at a loss.  I am on Mac. Post here when you figure it out please.

**Reply by gerroon** [reply]
Uninstalling and reinstalling BMO solved the issue.  I started using it since it came out, maybe updates were not reflected along the way.

**Comment by longy2k** [comment]
Please update to v2.1.1 and update the Ollama servers to bypass the CORS policy when using with Obsidian. Thanks!
